### PR TITLE
Small fixes to spring cloud config documentation

### PR DESCRIPTION
**Pull request by tanob**

### CODE REVIEW

File: docs/src/main/asciidoc/spring-cloud-config.adoc [hunk 1]
```diff
@@ -390,7 +390,8 @@ spring:
   cloud:
     config:
       server:
-        foo: bar
+        overrides:
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-config.adoc]: Why add this?

Comment (tanob) [inline docs/src/main/asciidoc/spring-cloud-config.adoc]: The documentation says: ```To declare overrides just add a map of name-value pairs to spring.cloud.config.server.overrides```. I thought the correct would be to have the overrides map inside the `overrides` key. Isn't that true?

**Comment by dsyer** [comment]
Thanks for spotting the error. I fixed it independently.

**Comment by tanob** [comment]
@dsyer there was also a typo fix in the PR

**Pull request closed without merging**